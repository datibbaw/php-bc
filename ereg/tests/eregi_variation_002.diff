006+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
006- Error: 8192 - Function eregi() is deprecated, %s(74)
010+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
010- Error: 8192 - Function eregi() is deprecated, %s(74)
014+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
014- Error: 8192 - Function eregi() is deprecated, %s(74)
018+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
018- Error: 8192 - Function eregi() is deprecated, %s(74)
022+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
022- Error: 8192 - Function eregi() is deprecated, %s(74)
026+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
026- Error: 8192 - Function eregi() is deprecated, %s(74)
030+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
030- Error: 8192 - Function eregi() is deprecated, %s(74)
034+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
034- Error: 8192 - Function eregi() is deprecated, %s(74)
038+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
039+ bool(false)
040+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)
041+ 
042+ Arg value Array 
043+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
044+ Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
045+ bool(false)
046+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)
047+ 
048+ Arg value Array 
049+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
050+ Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
051+ bool(false)
052+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)
053+ 
054+ Arg value Array 
055+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
056+ Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
057+ bool(false)
058+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)
059+ 
060+ Arg value Array 
061+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
062+ Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
038- Error: 8192 - Function eregi() is deprecated, %s(74)
043- Error: 8192 - Function eregi() is deprecated, %s(74)
044- Error: 2 - eregi() expects parameter 2 to be string, array given, %s(74)
045- NULL
046- Error: 8 - Array to string conversion, %seregi_variation_002.php(%d)
048- Arg value Array 
049- Error: 8192 - Function eregi() is deprecated, %s(74)
050- Error: 2 - eregi() expects parameter 2 to be string, array given, %s(74)
051- NULL
067+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
068+ Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
069+ bool(false)
070+ 
071+ Arg value  
072+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
073+ bool(false)
074+ 
075+ Arg value  
076+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
077+ bool(false)
078+ 
079+ Arg value 1 
080+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
081+ int(1)
082+ 
083+ Arg value  
084+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
085+ bool(false)
086+ 
087+ Arg value 1 
088+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
089+ int(1)
090+ 
091+ Arg value  
092+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
093+ bool(false)
094+ 
095+ Arg value  
096+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
097+ bool(false)
098+ 
099+ Arg value  
100+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
101+ bool(false)
102+ Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)
103+ 
104+ Arg value  
105+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
106+ Error: 2 - preg_match() expects parameter 2 to be string, object given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
107+ bool(false)
108+ 
109+ Arg value  
110+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
111+ bool(false)
113+ Arg value  
114+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
115+ bool(false)
116+ Done
052- Error: 8 - Array to string conversion, %seregi_variation_002.php(%d)
053- 
054- Arg value Array 
055- Error: 8192 - Function eregi() is deprecated, %s(74)
056- Error: 2 - eregi() expects parameter 2 to be string, array given, %s(74)
057- NULL
058- Error: 8 - Array to string conversion, %seregi_variation_002.php(%d)
059- 
060- Arg value Array 
061- Error: 8192 - Function eregi() is deprecated, %s(74)
062- Error: 2 - eregi() expects parameter 2 to be string, array given, %s(74)
063- NULL
064- Error: 8 - Array to string conversion, %seregi_variation_002.php(%d)
065- 
066- Arg value Array 
067- Error: 8192 - Function eregi() is deprecated, %s(74)
068- Error: 2 - eregi() expects parameter 2 to be string, array given, %s(74)
069- NULL
070- 
071- Arg value  
072- Error: 8192 - Function eregi() is deprecated, %s(74)
073- bool(false)
074- 
075- Arg value  
076- Error: 8192 - Function eregi() is deprecated, %s(74)
077- bool(false)
078- 
079- Arg value 1 
080- Error: 8192 - Function eregi() is deprecated, %s(74)
081- int(1)
082- 
083- Arg value  
084- Error: 8192 - Function eregi() is deprecated, %s(74)
085- bool(false)
086- 
087- Arg value 1 
088- Error: 8192 - Function eregi() is deprecated, %s(74)
089- int(1)
090- 
091- Arg value  
092- Error: 8192 - Function eregi() is deprecated, %s(74)
093- bool(false)
094- 
095- Arg value  
096- Error: 8192 - Function eregi() is deprecated, %s(74)
097- bool(false)
098- 
099- Arg value  
100- Error: 8192 - Function eregi() is deprecated, %s(74)
101- bool(false)
102- Error: 4096 - Object of class stdClass could not be converted to string, %s(73)
103- 
104- Arg value  
105- Error: 8192 - Function eregi() is deprecated, %s(74)
106- Error: 2 - eregi() expects parameter 2 to be string, object given, %s(74)
107- NULL
108- 
109- Arg value  
110- Error: 8192 - Function eregi() is deprecated, %s(74)
111- bool(false)
112- 
113- Arg value  
114- Error: 8192 - Function eregi() is deprecated, %s(74)
115- bool(false)
116- Done