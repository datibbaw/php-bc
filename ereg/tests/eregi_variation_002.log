
---- EXPECTED OUTPUT
*** Testing eregi() : usage variations ***
Error: 8 - Undefined variable: undefined_var, %s(64)
Error: 8 - Undefined variable: unset_var, %s(67)

Arg value 0 
Error: 8192 - Function eregi() is deprecated, %s(74)
bool(false)

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(74)
int(1)

Arg value 12345 
Error: 8192 - Function eregi() is deprecated, %s(74)
int(1)

Arg value -2345 
Error: 8192 - Function eregi() is deprecated, %s(74)
bool(false)

Arg value 10.5 
Error: 8192 - Function eregi() is deprecated, %s(74)
int(1)

Arg value -10.5 
Error: 8192 - Function eregi() is deprecated, %s(74)
int(1)

Arg value 101234567000 
Error: 8192 - Function eregi() is deprecated, %s(74)
int(1)

Arg value 1.07654321E-9 
Error: 8192 - Function eregi() is deprecated, %s(74)
int(1)

Arg value 0.5 
Error: 8192 - Function eregi() is deprecated, %s(74)
bool(false)
Error: 8 - Array to string conversion, %seregi_variation_002.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(74)
Error: 2 - eregi() expects parameter 2 to be string, array given, %s(74)
NULL
Error: 8 - Array to string conversion, %seregi_variation_002.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(74)
Error: 2 - eregi() expects parameter 2 to be string, array given, %s(74)
NULL
Error: 8 - Array to string conversion, %seregi_variation_002.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(74)
Error: 2 - eregi() expects parameter 2 to be string, array given, %s(74)
NULL
Error: 8 - Array to string conversion, %seregi_variation_002.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(74)
Error: 2 - eregi() expects parameter 2 to be string, array given, %s(74)
NULL
Error: 8 - Array to string conversion, %seregi_variation_002.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(74)
Error: 2 - eregi() expects parameter 2 to be string, array given, %s(74)
NULL

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(74)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(74)
bool(false)

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(74)
int(1)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(74)
bool(false)

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(74)
int(1)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(74)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(74)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(74)
bool(false)
Error: 4096 - Object of class stdClass could not be converted to string, %s(73)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(74)
Error: 2 - eregi() expects parameter 2 to be string, object given, %s(74)
NULL

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(74)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(74)
bool(false)
Done
---- ACTUAL OUTPUT
*** Testing eregi() : usage variations ***
Error: 8 - Undefined variable: undefined_var, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(64)
Error: 8 - Undefined variable: unset_var, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(67)

Arg value 0 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 12345 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value -2345 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 10.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value -10.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 101234567000 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 1.07654321E-9 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 0.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, object given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)
Done
---- FAILED
