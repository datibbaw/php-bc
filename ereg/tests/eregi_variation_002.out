*** Testing eregi() : usage variations ***
Error: 8 - Undefined variable: undefined_var, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(64)
Error: 8 - Undefined variable: unset_var, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(67)

Arg value 0 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 12345 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value -2345 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 10.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value -10.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 101234567000 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 1.07654321E-9 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 0.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_002.php(73)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - preg_match() expects parameter 2 to be string, object given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(110)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)
Done